### PR TITLE
chore(): 4.0.5

### PR DESCRIPTION
**Pull request by canonbrother**
#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: docs/node/CATEGORIES/08-account.md [hunk 1]
```diff
@@ -176,7 +181,15 @@ enum TransferDomainType {
...
   EVM = 3,
-};
+}
+
+enum TransferDomainOptionalKey {
+  SINGLE_KEY_CHECK = 'singlekeycheck',
+}
+
+interface TransferDomainOptionalInfo {
+  singleKeyCheck: true
```
Comment (fullstackninja864) [inline docs/node/CATEGORIES/08-account.md]:
```suggestion
  singleKeyCheck: boolean
```
it should be boolean ?

**Comment by sieniven** [comment]
singlekeycheck flag can be passed both on the RPC and/or set directly as a node config on start up. Reference: https://github.com/DeFiCh/ain/pull/2714 and https://github.com/DeFiCh/ain/pull/2709. To maintain backwards compatibility, a better fix could potentially be to pass the singlekeycheck flag directly on the node start up, which will default to skipping the singlekeycheck pipeline on transferdomain RPCs.

File: packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts [hunk 2]
```diff
@@ -233,7 +248,8 @@ describe('TransferDomain', () => {
             address: dvmAddr,
             amount: '3@DFI',
             domain: TransferDomainType.DVM
-          }
+          },
+          singlekeycheck: false as unknown as TransferDomainOptionalInfo
```
Comment (sieniven) [inline packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts]: singlekeycheck flag can be passed both on the RPC and/or set directly as a node config on start up. Reference: https://github.com/DeFiCh/ain/pull/2714 and https://github.com/DeFiCh/ain/pull/2709. To maintain backwards compatibility, a better fix could potentially be to pass the singlekeycheck flag directly on the node start up, which will default to skipping the singlekeycheck pipeline on transferdomain RPCs.

File: docs/node/CATEGORIES/08-account.md (hunk 1 above)
Comment (canonbrother) [reply docs/node/CATEGORIES/08-account.md]: true | false is boolean.. specify `true` means "true by default"
kind of
```
function get(item: any: isChecked: boolean)
vs
function get(item: any, isChecked: true)
```

File: packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts (hunk 2 above)
Comment (canonbrother) [reply packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts]: cleaner now
[7525d04](https://github.com/BirthdayResearch/jellyfishsdk/pull/2173/commits/7525d04f86ff7cf48d3ee9b971a214fa44289193)

hmm.. we still need show case how to TD with singlekeycheck params
so added a simple one [e7927cc](https://github.com/BirthdayResearch/jellyfishsdk/pull/2173/commits/e7927cc9842029559eee4c2763321936db1905a4)

File: docs/node/CATEGORIES/08-account.md (hunk 1 above)
Comment (canonbrother) [reply docs/node/CATEGORIES/08-account.md]: @fullstackninja864 
my bad.. that's not the right way on TS
as `Typescript - type 'false' is not assignable to type 'true'`...
[3df346c](https://github.com/BirthdayResearch/jellyfishsdk/pull/2173/commits/3df346c1ec939132dab28b6cfd995d3870380d27)

**Pull request merged**